requirements/base.in

```diff
@@ -3,3 +3,4 @@
 
 Django>=1.8,<3                       # Web application framework
 djangorestframework>=3.6
+edx-django-utils<3
```

Comment:
Please add a comment for the constraint.

Reply:
Also, we're now trying to keep such constraints in a separate `constraints.txt` file; see the latest OEP-18 revision for details: https://open-edx-proposals.readthedocs.io/en/latest/oep-0018-bp-python-dependencies.html#imposing-constraints-on-dependencies